projects/components/src/lib/directives/show-clipped-text.directive.ts

```diff
@@ -209,6 +209,23 @@ export class ShowClippedTextDirective implements OnDestroy, OnInit {
     @Input('vcdShowClippedTextMouseOutDelay')
     mouseoutDelay = 500;
 
+    @Input('vcdShowClippedTextDisabled')
+    set disabled(disabled: boolean) {
+        if (disabled && this.init) {
+            this.ngOnDestroy();
```

Comment:
I think we should have methods that initialize and remove the handlers instead of calling `ngOnDestroy` and `ngOnit` manually. They are only supposed to be called by the framework. Let's chat about it before you try to rework it